modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/SharedScopeTestWIthRestart.java

```diff
@@ -0,0 +1,128 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.am.integration.tests.other;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.clients.publisher.api.v1.dto.ScopeDTO;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
+import org.wso2.am.integration.tests.api.lifecycle.APIManagerLifecycleBaseTest;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SharedScopeTestWIthRestart extends APIManagerLifecycleBaseTest {
+    private final Log log = LogFactory.getLog(SharedScopeTestCase.class);
+
+    private String sharedScopeName = "TestSharedScopeWithRestart";
+    private String sharedScopeDisplayName = "Test Shared Scope with Restart";
+    private String description = "This is a test shared scope with Restart";
+    private String updatedDescription = "This is a updated test shared scope with Restart";
+    private String updatedDescription1 = "This is a updated test shared scope with Restart(2)";
+    private String updatedDescription2 = "This is a updated test shared scope with Restart(3)";
+    private List<String> roles = new ArrayList<>();
+    private String sharedScopeId;
+    private ServerConfigurationManager serverConfigurationManager;
+
+    @Factory(dataProvider = "userModeDataProvider")
+    public SharedScopeTestWIthRestart(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @DataProvider
+    public static Object[][] userModeDataProvider() {
+        return new Object[][] { new Object[] { TestUserMode.SUPER_TENANT_ADMIN }};
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init(userMode);
+        superTenantKeyManagerContext = new AutomationContext(APIMIntegrationConstants.AM_PRODUCT_GROUP_NAME,
+                APIMIntegrationConstants.AM_KEY_MANAGER_INSTANCE,
+                TestUserMode.SUPER_TENANT_ADMIN);
+        serverConfigurationManager = new ServerConfigurationManager(superTenantKeyManagerContext);
+        serverConfigurationManager.applyConfiguration(new File(
+                getAMResourceLocation() + File.separator + "operationPolicy"
```

Comment:
suggestion: Shall we move this configuration file to a new folder within `product-apim/modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/` ? Seems `operationPolicy` directory is meant to store policy related configs.